src/rust/cryptography-x509/src/certificate.rs

```diff
@@ -13,6 +14,12 @@ pub struct Certificate<'a> {
     pub signature: asn1::BitString<'a>,
 }
 
+impl<'a> Certificate<'a> {
+    pub fn extensions(&'a self) -> Result<Option<Extensions<'a>>, asn1::ObjectIdentifier> {
+        self.tbs_cert.extensions()
```

Comment:
Not sure it's worth it to have the method on TBS, is there a use case?

Reply:
That was mostly for `tbs_precertificate_bytes`, since we need `TbcCertificate.extensions()` in that context to perform the poison extension filtering. But maybe there's another way I should be getting the extensions there?

Reply:
Ah yep, I can get rid of this entirely.